src/application/controller/UserController.ts

```diff
@@ -0,0 +1,153 @@
+import { BaseController } from './BaseController';
+import { NextFunction, Request, Response } from 'express';
+import { MongoAdapter } from '../../infrastructure/MongoAdapter';
+import { UserRepository } from '../../infrastructure/repository/UserRepository';
+import {operations} from "../../interface/api"
+
+class UserController extends BaseController {
+
+  async getUsers(req: Request, res: Response) {
+    const mongoAdapter = MongoAdapter.getInstance();
+
+    const userRepo = new UserRepository(mongoAdapter, 'userDetails');
+    await userRepo.isConnected();
+
+    var query = {};
+
+    // const queryParams = req.query;
```

Comment:
Remove commented out code

Reply:
possibly useful for developers down the line